monitoring/mongodb/dashboard.json

```diff
@@ -1084,11 +1073,11 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "mongodb_mongod_replset_oplog_size_bytes{pod=~\"${mongod}\", namespace=\"${namespace}\"}",
+          "expr": "mongodb_oplog_stats_size{pod=~\"${mongod}\", namespace=\"${namespace}\"}",
           "format": "time_series",
           "interval": "",
           "intervalFactor": 2,
-          "legendFormat": "{{type}}",
+          "legendFormat": "{{pod}}",
```

Comment:
the "pod" may vary between members of replset, this may look very strange...
is there a way to get the replset name instead?